cfunc/Makefile

```diff
@@ -1,8 +1,11 @@
-CC = g++
+
+CC := $(if $(CXX),$(CXX), g++)
 
 INCLUDE_DIR = include
 BUILD_DIR = bin
-COREIR_DIR = /nobackup/joeyliu/aha/coreir
+COREIR_DIR := $(if $(COREIR_DIR),$(COREIR_DIR),/nobackup/joeyliu/aha/coreir)
```

Comment:
Can we changes this to 
COREIR_DIR ?= ../../coreir

Reply:
@jeffsetter just changed it.

Reply:
Can we change it to ../../coreir
instead of Joey's coreir folder?

Reply:
So do I need to change to ../../coreir to pass the travis?

Reply:
Not for travis (since it will likely defined COREIR_DIR), but it would be best to change it for anyone that clones this repo.